contracts/AnchoredView/SymbolConfiguration.sol

```diff
@@ -49,7 +51,7 @@ contract SymbolConfiguration {
     bytes32 constant symbolUsdt = keccak256(abi.encodePacked("USDT"));
 
     //  Address of the underlying tokens
-    address public immutable ethAddress = 0xe;
+    address public constant ethAddress = address(1);
```

Comment:
I feel like `ethUnderlying` should be `address(0)` since its not even a valid address